service/matching/handler/engine.go

```diff
@@ -249,15 +249,14 @@ func (e *matchingEngineImpl) getTaskListByDomainLocked(domainID string) *types.G
 	decisionTaskListMap := make(map[string]*types.DescribeTaskListResponse)
 	activityTaskListMap := make(map[string]*types.DescribeTaskListResponse)
 	for tl, tlm := range e.taskLists {
-		if tlm.GetTaskListKind() == types.TaskListKindNormal && tl.GetDomainID() == domainID {
+		if tl.GetDomainID() == domainID && tlm.GetTaskListKind() == types.TaskListKindNormal {
 			if types.TaskListType(tl.GetType()) == types.TaskListTypeDecision {
 				decisionTaskListMap[tl.GetRoot()] = tlm.DescribeTaskList(false)
+			} else {
+				activityTaskListMap[tl.GetRoot()] = tlm.DescribeTaskList(false)
```

Comment:
This looks like a bug fix. `activityTaskListMap` was populated with tasklist of decision type. Can you update PR description to make this more explicit?